packages/cli/src/telemetry/telemetry-event-relay.service.ts

```diff
@@ -101,6 +108,16 @@ export class TelemetryEventRelay {
 		this.eventService.on('login-failed-due-to-ldap-disabled', (event) => {
 			this.loginFailedDueToLdapDisabled(event);
 		});
+
+		this.eventService.on('workflow-created', (event) => {
+			this.workflowCreated(event);
+		});
+		this.eventService.on('workflow-deleted', (event) => {
+			this.workflowDeleted(event);
+		});
+		this.eventService.on('workflow-saved', async (event) => {
+			await this.workflowSaved(event);
```

Comment:
Looks like we have Sentry's `OnUnhandledRejection` integration enabled, so having a dangling promise chain is not causing us any issues